### PR TITLE
Configure settings.js as external

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Release candidate for 4 December.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
         <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
         <link rel="stylesheet" href="/styles/main.css">
+        <script src="/settings.js" async=false></script>
```
Comment (philschatz) [inline src/index.html]: can you include quotes around `false` for consistency?

File: script/build [hunk 2]
```diff
@@ -3,5 +3,3 @@ cd "$(dirname "$0")/.." || exit 111
 source ./script/bootstrap || exit 111
 
 npm run-script build || exit 2
-
-"$(npm bin)/webpack" --config webpack.config.js
```
Comment (philschatz) [inline script/build]: doh! I don't know why this was still in here

**Comment by RoyEJohnson** [comment]
@philschatz Tests are failing because they can't find `settings`. Does that mean a change needs to be made in the build phase?

**Comment by philschatz** [comment]
It seems Jest needs a module mapping to the `settings.js.example` (or rename it to `settings.example.js` so Jest can `require(...)` it)

**Pull request merged**